pkg/controller/controller_utils.go

```diff
@@ -420,3 +421,21 @@ func FilterActivePods(pods []api.Pod) []*api.Pod {
 	}
 	return result
 }
+
+func SyncAllPodsWithStore(kubeClient client.Interface, store cache.Store) {
+	var allPods *api.PodList
+	var err error
+	listOptions := api.ListOptions{LabelSelector: labels.Everything(), FieldSelector: fields.Everything()}
+	for {
+		if allPods, err = kubeClient.Pods(api.NamespaceAll).List(listOptions); err != nil {
```

Comment:
The controller has one, too (https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/framework/controller.go#L100) but it doesn't tell you what you want, either.

I think what you're doing may actually be appropriate. It does affect the initial batch of events you get from the informer; instead of a bunch of OnAdd you'll get a bunch of OnUpdates. Can you add this to the function comment? Actually if you could make the function comment tell you why you would want to call it instead of (or in addition to) what it does, that would be helpful.

If we kept local state, it would make sense to load here the last set of pods we knew about.